test/e2e/rule_test.go

```diff
@@ -97,6 +97,34 @@ func reloadRulesHTTP(t *testing.T, ctx context.Context, endpoint string) {
 	testutil.Equals(t, 200, resp.StatusCode)
 }
 
+func reloadRulesSignal(t *testing.T, r *e2ethanos.Service) {
+	c := e2e.NewCommand("kill", "-1", "1")
+	_, _, err := r.Exec(c)
+	testutil.Ok(t, err)
+}
+
+func checkReloadSuccessful(t *testing.T, ctx context.Context, endpoint string) {
+	req, err := http.NewRequestWithContext(ctx, "GET", "http://"+endpoint+"/api/v1/rules", ioutil.NopCloser(bytes.NewReader(nil)))
+	testutil.Ok(t, err)
+	resp, err := http.DefaultClient.Do(req)
+	testutil.Ok(t, err)
+	testutil.Equals(t, 200, resp.StatusCode)
+	defer resp.Body.Close()
```

Comment:
Let's just do: `testutil.Ok(t, resp.Body.Close())`

Reply:
@jmichalek132 could you please simply do `testutil.Ok(t, resp.Body.Close())` after `ioutil.ReadAll()`? It will be simpler + you'll get a failure if `Close()` fails

Reply:
Like this?